client/src/containers/app.coffee

```diff
@@ -17,10 +18,11 @@ App = React.createClass
     loginActions = bindActionCreators LoginActions, dispatch
     pageLocationActions = bindActionCreators PageLocationActions, dispatch
     repoActions = bindActionCreators RepoActions, dispatch
+    registerActions = bindActionCreators RegisterActions, dispatch
 
     div {},
       Header {login, loginActions, pageLocationActions}
-      MainSection {login, repo, repoActions, pageLocation, loginActions, pageLocationActions}
+      MainSection {login, repo, repoActions, pageLocation, loginActions, pageLocationActions, registerActions}
```

Comment:
I mentioned this already, but what would be a good way to not pass all the actionCreators down through props?